server/routes/amendAReferral/employment-responsibilities/amendEmploymentResponsibilitiesPresenter.ts

```diff
@@ -0,0 +1,53 @@
+import { FormValidationError } from '../../../utils/formValidationError'
+import PresenterUtils from '../../../utils/presenterUtils'
+import SentReferral from '../../../models/sentReferral'
+
+export default class AmendEmploymentResponsibilitiesPresenter {
+  private readonly utils = new PresenterUtils(this.userInputData)
+
+  readonly errorMessage = PresenterUtils.errorMessage(this.error, 'abc') // AmendMaximumEnforceableDaysForm.reasonForChangeId)
+
+  readonly errorSummary = PresenterUtils.errorSummary(this.error)
+
+  readonly backLinkUrl: string
+
+  constructor(
+    private readonly sentReferral: SentReferral,
+    private readonly error: FormValidationError | null = null,
+    private readonly userInputData: Record<string, string> | null = null,
+    readonly showNoChangesBanner: boolean = false
+  ) {
+    this.backLinkUrl = `/probation-practitioner/referrals/${sentReferral.id}/details`
+  }
+
+  private errorMessageForField(field: string): string | null {
+    return PresenterUtils.errorMessage(this.error, field)
+  }
+
+  readonly text = {
+    responsibilities: {
+      title: `Do you want to change whether ${this.sentReferral.referral.serviceUser?.firstName} has caring or employment responsibilities?`,
+      hasAdditionalResponsibilities: {
+        label: `Do you want to change whether ${this.sentReferral.referral.serviceUser?.firstName} has caring or employment responsibilities?`,
+        hint: 'For example, times and dates when they are at work.',
+        errorMessage: this.errorMessageForField('has-additional-responsibilities'),
+      },
+      whenUnavailable: {
+        label: `Provide details of when ${this.sentReferral.referral.serviceUser?.firstName} will not be able to attend sessions`,
```

Comment:
In the other amend pages, we are using serviceUser.firstName which I heard the more reliable way to grab the username

Reply:
is it worth the extra call through community api for something we already have access to this way?